### PR TITLE
Grokkin' the Docs: Experiences of a New Contributor blog post

### LINKED ISSUE #15965: Create blog post on new contributor to k8s and SIG Docs experience
**This is a Feature Request**

**What would you like to be added**
Chris Anisczyk would like me to write a blog post on my experience as new contributor to SIG Docs. 

**Why is this needed**
My supervisor's supervisor requested it.

**Comments**

### PR DESCRIPTION
**Pull request by aimeeu**
This is a new PR based on #16414.  GitHub issues  prevented Zach from submitting his review.

Zach Corleissen asked me to write a blog post on my first few months as a new SIG Docs contributor.
The CNCF would like this post published the first week in November. My goal is Wed, 11/6.

Fixes #15965

**Preview**:  https://deploy-preview-17314--kubernetes-io-master-staging.netlify.com/blog/2019/11/06/grokkin-the-docs/

Signed-off-by: Aimee Ukasick <aimeeu.opensource@gmail.com>

### CODE REVIEW

**Comment by tengqm** [comment]
This is the best blog I read on documentation!
/LGTM

File: content/en/blog/_posts/grokkin-the-docs.md [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+[page](https://github.com/kubernetes/community/blob/master/sig-list.md) helps,
+but it is not enough. At a page level in the documentation, it's not
+always obvious which SIG or project has domain expertise. The page's front
+matter sometimes list reviewers but never lists a SIG or project. Each page should
+indicate who is responsible for content, so that SIG Docs triagers know where to
+route issues.
+
+## Observation 08: SIG Docs is understaffed {#observation-8}
+
+Documentation is the #1 driver of software adoption<sup>1</sup>.
```
Comment (zacharysarah) [inline content/en/blog/_posts/grokkin-the-docs.md]:
```suggestion
Documentation is the number one driver of software adoption<sup>1</sup>.
```

File: content/en/blog/_posts/grokkin-the-docs.md [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+Yet few companies are willing to allocate tech writers at least
```
Comment (zacharysarah) [inline content/en/blog/_posts/grokkin-the-docs.md]: Hopefully they _are_ willing!

```suggestion
Yet few companies have hired tech writers at least
```

File: content/en/blog/_posts/grokkin-the-docs.md [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+half-time to Kubernetes. Many contributors are able to devote a small amount of
+time to SIG Docs but few are full-time. Of the people __reviewing and
+approving__ English language documentation, probably only one or two are
+allocated full-time. Of the people __contributing content__ to the English
+language documentation, probably less than a handful are allocated full-time.
+That's very disheartening for online documentation that has had over 53 million
+unique page views from users in 229 countries year-to-date in 2019.
```
Comment (zacharysarah) [inline content/en/blog/_posts/grokkin-the-docs.md]: I 💔 hyphens.

```suggestion
unique page views from users in 229 countries year to date in 2019.  
```

**Comment by kbhawkey** [comment]
Trying to read the preview. Is this the preview?
https://deploy-preview-17314--kubernetes-io-master-staging.netlify.com/blog/

**Comment by aimeeu** [comment]
> Trying to read the preview. Is this the preview?
> https://deploy-preview-17314--kubernetes-io-master-staging.netlify.com/blog/

No it's not. Hmmmmm.  Oh wait! The publish date is still 9/4 (this blog has been under review for a looooooong time).  
https://deploy-preview-17314--kubernetes-io-master-staging.netlify.com/blog/2019/09/04/grokkin-the-docs/

File: content/en/blog/_posts/grokkin-the-docs.md (hunk 3 above)
Comment (aimeeu) [reply content/en/blog/_posts/grokkin-the-docs.md]: Either is correct, but I bow to the wandering editor's red pen.  lol

File: content/en/blog/_posts/grokkin-the-docs.md (hunk 2 above)
Comment (aimeeu) [reply content/en/blog/_posts/grokkin-the-docs.md]: For the life of me I could not remember what you replaced this paragraph with during our meeting.

File: content/en/blog/_posts/grokkin-the-docs.md (hunk 1 above)
Comment (aimeeu) [reply content/en/blog/_posts/grokkin-the-docs.md]: Done.

**Comment by aimeeu** [comment]
/assign @kbarnard10  
as blog approver

File: content/en/blog/_posts/grokkin-the-docs.md (hunk 2 above)
Comment (zacharysarah) [reply content/en/blog/_posts/grokkin-the-docs.md]: I can't remember, either. 😞

Comment (aimeeu) [reply content/en/blog/_posts/grokkin-the-docs.md]: I did remove the "Of people..." paragraphs. Let me know what you think.

**Comment by zacharysarah** [comment]
Leaving `/approve` to @kbarnard10.

/lgtm

**Comment by zacharysarah** [comment]
Thanks, @kbarnard10!

**Pull request merged**